common/lib/xmodule/xmodule/templates/problem/checkboxes_response.yaml

```diff
@@ -1,24 +1,60 @@
 ---
 metadata:
-    display_name: Checkboxes
+    display_name: Checkbox
     markdown: |
-      A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.
-      >>Select the answer that matches<<
+       To create a checkbox problem in the simple editor:
+       - Enter the text of the main question between angle brackets.
+       - Add brackets that contain a space before each answer option: [ ]
+       - For each correct answer, add an x between the parentheses: [x] 
+         Note that a problem can have more than one correct answer.
+       - To add an explanation, enter the text of the explanation, and then highlight the explanation and click the light bulb icon.
+       You can replace the text in the following example problem with your own text.
+       _____________________________________________________________________________
 
-      [x] correct
-      [ ] incorrect
-      [x] correct
+      >>The following languages  are in the Indo-European family:<<
+
+      [x] Urdu
+      [ ] Finnish
+      [x] Marathi
+      [x] French
+      [ ] Hungarian
+
+      Note: Make sure you select all of the correct options—there may be more than one!
+
+      [explanation]
+      Urdu, Marathi, and French are all Indo-European languages, while Finnish and  Hungarian are in the Uralic family.
+      [explanation]
 
 data: |
       <problem>
-        <p>A checkboxes problem presents checkbox buttons for student input. Students can select more than one option presented.</p>
-        <p>Select the answer that matches</p>
+      <p>To create a checkbox problem in the simple editor:</p>
+      <p>- Enter the text of the main question between angle brackets.</p>
+      <p>- Add brackets that contain a space before each answer option: [ ]</p>
+      <p>- For each correct answer, add an x between the parentheses: [x] </p>
+      <p>Note that a problem can have more than one correct answer.</p>
+      <p>- To add an explanation, enter the text of the explanation, and then highlight the explanation and click the light bulb icon.</p>
+      <p>You can replace the text in the following example problem with your own text.</p>
+      <p>_____________________________________________________________________________</p>
+
+      <p>The following languages  are in the Indo-European family:</p>
```

Comment:
@srpearce, for some background, @ahsan-ul-haq is converting these problem types to use the best practice and better supported technique of a using a ```<fieldset>``` and ```<legend>``` combination rather than a ```<p>``` and ```<div role=group aria-label="[repeated text">``` combo.  The latter forces the screen reader user to either hear the associated question text twice or not at all.  I'm wondering if we should update this example to use the better supported format.

*in place of the ```<p>``` here:
```
<fieldset>
<legend>The following languages are in the Indo-European family:</legend>
<checkboxgroup direction="vertical">
```

You can see that the ```label``` attribute is no longer needed on the checkboxgroup element because there is a programmatic relationship between the ```<legend>``` and the checkbox/radio button elements.